### PR TITLE
GitHub workflow fix

### PR DESCRIPTION
**Pull request by maxpaj**

### CODE REVIEW

**Comment by FrimJo** [comment]
Does not look like you are respecting `.prettierrc.json` 😋 See a lot of added `;` for example.

File: remix.init/setup-git-repository.js [hunk 1]
```diff
@@ -1,30 +1,62 @@
...
+  });
+
+  execSync(
+    `GH_DEBUG=1 gh repo create ${appName} --private --push --source ${rootDirectory}`,
+    {
+      cwd: rootDirectory,
+    }
+  );
+
+  execSync(`azd pipeline config`, {
```
Comment (FrimJo) [inline remix.init/setup-git-repository.js]: According to [Authorize GitHub to deploy to Azure](https://learn.microsoft.com/en-us/azure/developer/azure-developer-cli/configure-devops-pipeline?tabs=GitHub#authorize-github-to-deploy-to-azure): `azd pipeline config` also creates a private GitHub repository and pushes code to the new repo. Did we figure out what we need to to using `gh` command? Or should everything done using `azd pipeline config`?

Comment (maxpaj) [reply remix.init/setup-git-repository.js]: It actually won't if we're already in a Git repo!

**Comment by maxpaj** [comment]
Can't get my IDE to respect prettier rules, not sure why.

File: remix.init/setup-git-repository.js (hunk 1 above)
Comment (maxpaj) [reply remix.init/setup-git-repository.js]: Removed the `gh` stuff completely. `azd pipeline config` and such takes care of that setup for us.

**Comment by FrimJo** [comment]
@maxpaj I've rebased on master after updating prettier config and added workspace default settings

**Review by FrimJo — APPROVED** [review]

**Pull request merged**